src/CommunityToolkit.Maui.Camera/Views/CameraView.shared.cs

```
@@ -230,53 +230,47 @@ protected virtual void Dispose(bool disposing)
 		}
 	}
 
-	static Command<CancellationToken> CreateCaptureImageCommand(BindableObject bindable)
+	Command<CancellationToken> CreateCaptureImageCommand()
 	{
-		var cameraView = (CameraView)bindable;
-		return new(async token => await cameraView.CaptureImage(token).ConfigureAwait(false));
+		return new(async token => await CaptureImage(token).ConfigureAwait(false));
 	}
 
-	static Command<CancellationToken> CreateStartCameraPreviewCommand(BindableObject bindable)
+	Command<CancellationToken> CreateStartCameraPreviewCommand()
 	{
-		var cameraView = (CameraView)bindable;
-		return new(async token => await cameraView.StartCameraPreview(token).ConfigureAwait(false));
+		return new(async token => await StartCameraPreview(token).ConfigureAwait(false));
 	}
 
-	static Command CreateStopCameraPreviewCommand(BindableObject bindable)
+	Command CreateStopCameraPreviewCommand()
 	{
-		var cameraView = (CameraView)bindable;
-		return new(_ => cameraView.StopCameraPreview());
+		return new(_ => StopCameraPreview());
```

`_ =>` will swallow exceptions. Switch to capturing the exception and maybe show a log message.

---

The `_ =>` was already in the original code and remains unchanged. The only update here was converting the method from static to non-static, so the behavior should be identical to before.